Assets/MixedRealityToolkit.SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -0,0 +1,2232 @@
+                    Debug.Log(name + " UseOnPreRender is enabled, but ScrollingObjectCollection needs UpdateCollection() to be called first.");
+                    return;
+                }
+
+                clipBox.UseOnPreRender = true;
+
+                //Subscribe to the preRender callback on the main camera so we can intercept it and make sure we catch
+                //any dynamically created children in our list
+                cameraMethods = CameraCache.Main.gameObject.EnsureComponent<CameraEventRouter>();
+                cameraMethods.OnCameraPreRender += OnCameraPreRender;
+            }
+        }
+
+        private void Start()
+        {
+            if (setUpAtRuntime)
+            {
+                UpdateCollection();
+            }
+        }
+
+        private void Update()
+        {
+            //early bail, the component isn't set up properly
+            if (scrollContainer == null) { return; }
+
+            bool nodeLengthCheck = NodeList.Count > (ViewableArea * Tiers);
+
+            // Force the position if the total number of items in the list is less than the scrollable area
+            if (!nodeLengthCheck)
+            {
+                workingScrollerPos = Vector3.zero;
+                ApplyPosition(workingScrollerPos);
+            }
+
+            //The scroller has detected input and has a valid pointer
+
+            if (isEngaged && TryGetPointerPositionOnPlane(out Vector3 currentPointerPos))
+            {
+                Vector3 handDelta = initialPointerPos - currentPointerPos;
+                handDelta = transform.InverseTransformDirection(handDelta);
+
+                //Lets see if this is gonna be a click or a drag
+                //Check the scroller's length state to prevent resetting calculation
+                if (!isDragging && nodeLengthCheck)
+                {
+                    //grab the delta value we care about
+                    float absAxisHandDelta = (scrollDirection == ScrollDirectionType.UpAndDown) ? Mathf.Abs(handDelta.y) : Mathf.Abs(handDelta.x);
+
+                    //Catch an intentional finger in scroller to stop momentum, this isn't a drag its definitely a stop
+                    if (absAxisHandDelta > (handDeltaMagThreshold * 0.1f) || TimeTest(initialPressTime, Time.time, dragTimeThreshold))
+                    {
+                        scrollVelocity = 0.0f;
+                        avgVelocity = 0.0f;
+
+                        isDragging = true;
+                        velocityState = VelocityState.None;
+
+                        //now that we're dragging, reset the interacted with interactable if it exists
+                        Interactable ixable = initialFocusedObject.GetComponent<Interactable>();
+                        if(ixable != null)
+                        {
+                           ixable.ResetAllStates();
+                        }
+
+                        //TODO: Reset the state of PressableButton, when possible. 
```

Comment:
TODO?

Reply:
This isn't supported by PressableButtons YET... @thalbern probably a UX task for the next release 😄

Reply:
Ok, well at least file a bug/issue so @thalbern and others can track etc